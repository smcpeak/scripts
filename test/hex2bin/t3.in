4141 9 41