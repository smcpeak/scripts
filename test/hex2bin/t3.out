41 41 09 41
